Composer/packages/client/src/constants.ts

```diff
@@ -227,7 +227,7 @@ export const DialogCreationCopy = {
   get SELECT_LOCATION() {
     return {
       title: formatMessage('Select a Bot'),
-      subText: formatMessage('Which bot do you want to open?'),
+      subText: formatMessage('Specify an existing bot to connect to your Azure Bot resource.'),
```

Comment:
do we want to change the text for all the scenarios? what if a user is just simply opening a bot?

Reply:
@Andrea-Orimoto Should we change the text for all the scenarios?

Reply:
We should not change the text for all scenarios. It's important that we preserve the copy for scenarios that are not the ABS Handoff scenario

Reply:
@cwhitten Sure, I will fix it today.